README.md

```diff
@@ -71,14 +71,14 @@ groups:
 
 * PEADM is compatible with Puppet Enterprise 2019.8.1 or newer versions.
 * To use PEADM, you must first [install Bolt](https://www.puppet.com/docs/bolt/latest/bolt_installing) version 3.17.0 or newer.
-* PEADM supports PE installations on the following operating systems:
+* PEADM can be used on the following operating systems:
```

Comment:
Does this change intend to hide the difference between the platform PE is installed onto, and the platform PEADM is installed onto? Example: PEADM should work on Windows, while PE infrastructure nodes do not.

Reply:
The change is intended to differentiate between the platforms PEADM can work on versus the platforms that PE can be installed on (which are listed in version-specific PE docs.

If this list of PEADM-compatible platforms is not up to date (e.g. from your comment I guess some Windows versions are missing), we should add any platforms that are missing.

Reply:
So the list below "PEADM can be used on" should include MacOS and Windows?

Reply:
Yes. Here, I think we should have a complete and up to date list of the OS platforms that PEADM works on. I wondered, for example, about Ubuntu 22, SLES 15 and some of the other platforms that are currently PE-supported.

Reply:
Basically, PEADM should work on all operating systems that support Bolt. Can't we say that?

Reply:
Right. That makes sense! I wonder if the list here is up to date: 
https://www.puppet.com/docs/bolt/latest/bolt_installing

Reply:
Updated with a general reference to Bolt supported OS's.